src/test/scala/org/kibanaLoadTest/helpers/ESClient.scala

```diff
@@ -47,18 +49,27 @@ class ESClient(config: ESConfiguration) {
 
     try {
       logger.info(s"Ingesting to stats cluster: ${jsonList.size} docs")
-      jsonList.par.foreach(json => {
-        try {
-          client.index(
+
+      val it = jsonList.sliding(BULK_SIZE, BULK_SIZE)
```

Comment:
```suggestion
      val it = jsonList.grouped(BULK_SIZE)
```
Don't quote me, and of course, this is not a demand, but it would have less duplication.
If you have to push again, it could be useful.  If not re-pushing for any other reason...meh lol

My silly comment aside, it looks like `.grouped` and `.sliding` have the same performance.
So, `.sliding` gives you some flexibility, but I'm not sure we need that flexibility.

I read about it here: https://stackoverflow.com/questions/32874419/scala-slidingn-n-vs-groupedn

Reply:
Nice point, I was not aware of it :)